source/reference/modules/engine/index.rst

```diff
@@ -1330,6 +1330,18 @@ Crafter Engine sports a built-in cache engine with an LRU (least recently used)
 
 .. note:: When running in Preview Mode (inside Studio for preview purposes), Crafter Engine's cache is disabled to help authors see their changes immediately.
 
+Engine has multiple caches:
+
+.. version_tag::
+    :label: Since
+    :version: 4.3.1
+
+- **user** cache for objects cached by Groovy code.
+  This cache is cleared manually by calling the :base_url:`cacheClear <_static/api/engine.html#tag/cache/operation/cacheClear>` API.
```

Comment:
Both cache's use the same API to clear and to get statistics, but with a different parameter, called `cacheType`, which can be `system` or `application`.